packages/agent/src/agents/index.ts

```
@@ -163,6 +164,7 @@ export class AgentSystem {
         modelSelector: modelSelector,
       };
 
+      const controller = new AbortController();
```

ESLint

[error] 'controller' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)